src/sampleData/showcase_projects.json

```diff
@@ -40,5 +40,26 @@
         "permalink": "?lon=8.59559&lat=46.69875&elevation=417&heading=21&pitch=23&map=ch.swisstopo.swissimage&map_transparency=0.43&layers=ch.swisstopo.swisstlm3d.3d,ch.swisstopo.geologie-geocover,water_tunnel,rail_tunnel,road_tunnel&layers_visibility=true,false,true,true,true&layers_transparency=0.00,0.82,0.00,0.00,0.000&target={\"lon\":\"8.60072\",\"lat\":\"46.70794\",\"height\":881.8231693363072}"
       }
     ]
+  },
+  {
+    "title": "Water Tower Switzerland",
+    "description": "Hydropower is responsible for generating around 57 % of Switzerland’s electricity. Like veins, countless water tunnels and conduit shafts criss-cross the Alps, channelling the precious resource with the aim of producing energy.",
+    "created": "10 am January 2022",
```

Comment:
Can you correct that to "10 January 2022"? It's wrong also in the previous projects... It was a confusion between date and time ("am" / "pm") and German ("Erstellt am...").

If it is easy, change it also for the other old projects listed above!

The "created" is currently just a text, you don't have to follow a specific format. (maybe we have to change it one day, but let's handle it like that for the moment).

Reply:
Time changed. I also did not get the point about the "am". I removed it also for the other dashboard topics.